packages/x-data-grid-premium/src/tests/aggregation.DataGridPremium.test.tsx

```diff
@@ -409,8 +409,8 @@ describe('<DataGridPremium /> - Aggregation', () => {
 
       act(() => apiRef.current.showColumnMenu('id'));
```

Comment:
Turning these async can also be done in follow up.

Reply:
Do you mean adding an `await` before `act`?
The plugin didn't complain about this. 🙈 🤷

Reply:
> Do you mean adding an await before act?

also making the function `async`

> The plugin didn't complain about this. 🙈 🤷

...yet 😄 https://github.com/testing-library/eslint-plugin-testing-library/issues/915

More info in [this issue](https://github.com/mui/mui-public/issues/174)

Reply:
Yeah... Well, it looks like it would be great to get the rule and then update the tests with that rule in action. 🙈